Makefile

```diff
@@ -1,7 +1,6 @@
-langNames.db: language_names/scraped.sql language_names/scraped-sil.sql language_names/manual.sql language_names/variants.sql
+langNames.db: langNames.db
```

Comment:
The prerequisites still need to include all the TSV files.

Reply:
I don't think you're understanding how a Makefile works. I suggest reading up on them a bit. It doesn't make any sense for a target to have itself as a prerequisite.

Reply:
Yeah I was thinking the same way. Sorry for this change, I will make this right.